docs/tutorials/multimodal/beginner_text.md

```diff
@@ -87,17 +87,17 @@ And you can easily obtain predictions from these models by calling `predictor.pr
 sentence1 = "it's a charming and often affecting journey."
 sentence2 = "It's slow, very, very, very slow."
 predictions = predictor.predict({'sentence': [sentence1, sentence2]})
-print('"Sentence":', sentence1, '"Predicted Sentiment":', predictions.iloc[0])
-print('"Sentence":', sentence2, '"Predicted Sentiment":', predictions.iloc[1])
+print('"Sentence":', sentence1, '"Predicted Sentiment":', predictions[0])
```

Comment:
iloc[0] means to take the 0-th element. Is there a specific reason for this change?

Reply:
To accommodate the zero-shot tutorial needs, `predict()` returns a numpy array if the input data is in the dictionary or list format. It returns a dataframe if the input data is a dataframe or users specify `as_pandas=True`.

Reply:
I see. Will it make sense to just change the default of "as_pandas"? Or we can leave it to a later PR.

Reply:
After a 2nd thought, I think the current design of having as_pandas=None by default also makes sense.

Reply:
The `predictions ` in the above case is a ndarray since the input is a dictionary. It is necessary to remove `iloc` here.